eZ/Publish/API/Repository/BookmarkService.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\API\Repository;
+
+use eZ\Publish\API\Repository\Values\Bookmark\BookmarkList;
+use eZ\Publish\API\Repository\Values\Content\Location;
+
+/**
+ * Bookmark Service.
+ */
+interface BookmarkService
+{
+    /**
+     * Add location to bookmarks.
+     *
+     * @param \eZ\Publish\API\Repository\Values\Content\Location $location
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException When location is already bookmarked
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     */
+    public function createBookmark(Location $location): void;
+
+    /**
+     * Removes given location from bookmarks.
```

Comment:
Let's keep language consistent, so `Remove` as above you have `Add` (which I think is a correct form of documenting methods, though in the past we didn't care too much).

Reply:
As I see `Deletes` is used more often too.